packages/viewer/src/components/viewer-intersection-query-tool/viewer-intersection-query-tool.tsx

```diff
@@ -0,0 +1,170 @@
+import { Component, Element, h, Host, Prop, State, Watch } from '@stencil/core';
+import { Disposable } from '@vertexvis/utils';
+
+import { VolumeIntersectionQueryController } from '../../lib/volume-intersection/controller';
+import { VolumeIntersectionQueryInteractionHandler } from '../../lib/volume-intersection/interactions';
+import {
+  QueryType,
+  VolumeIntersectionQueryDetails,
+  VolumeIntersectionQueryModel,
+} from '../../lib/volume-intersection/model';
+
+export type DefaultOperationType = 'select' | 'deselect';
+
+@Component({
+  tag: 'vertex-viewer-intersection-query-tool',
+  styleUrl: 'viewer-intersection-query-tool.css',
+  shadow: true,
+})
+export class ViewerIntersectionQueryTool {
```

Comment:
Would it be too specific to name this thing `BoxQueryTool`? Or something like that?

Reply:
Yeah, I think it's worth considering some alternate names for this tool. I do like `box-query-tool` a bit better, so I'll update it!